pkg/antctl/handlers/agentinfo.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/monitor"
+)
+
+var _ Factory = new(AgentInfo)
+
+// ComponentAgentInfoResponse describes the internal response struct of agent-info command.
+// It only contains agent's node subnet and connection status of controller, OVSDB and openflow.
+type ComponentAgentInfoResponse struct {
+	ControllerConn string `json:"controllerConn,omitempty" yaml:"controllerConn,omitempty"`
+	OVSDBConn      string `json:"ovsdbConn,omitempty" yaml:"ovsdbConn,omitempty"`
```

Comment:
Do you think we can do something like this, which is more in line with the K8s style for serializable structs:
```
type ConnStatus string

const (
	ConnStatusUp ConnStatus = "UP"
	ConnStatusDown ConnStatus = "DOWN"
)

type ComponentAgentInfoResponse struct {
	ControllerConn ConnStatus `json:"controllerConn,omitempty" yaml:"controllerConn,omitempty"`
        // ...
}
```
This makes it clear that the "*Conn" fields can only be set to "UP" or "DOWN" and helps document the response IMO.

Reply:
Yes, it is better. I have updated the code. I put ConnStatus at package monitor because maybe other cli will need this.